x-pack/plugins/viz_editor/public/common/lib/field_config/field_config.ts

```diff
@@ -4,10 +4,9 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { SelectOperator } from '../../../../common';
-import { Field } from '../vis_model';
+import { DatasourceField, SelectOperator } from '../../../../common';
 
-export function getOperationsForField(field: Field): SelectOperator[] {
+export function getOperationsForField(field: DatasourceField): SelectOperator[] {
   // TODO: Make this configuration plugin-oriented
```

Comment:
To implement the suggestions from Alona, we have to differentiate between x and y axis here. Will we do this on a separate PR?

Reply:
Yes, definitely a separate PR for improving suggestions